dashboard/app/controllers/api/v1/sections_controller.rb

```diff
@@ -164,6 +164,15 @@ def valid_scripts
     render json: scripts
   end
 
+  # GET /api/v1/sections/require_captcha
+  # Get the recaptcha site key for frontend and whether current user requires captcha verification
+  def require_captcha
+    return head :forbidden unless current_user
+    site_key = CDO.recaptcha_site_key
+    # TODO: (dbeckmen) debug and insert server side logic in user model for tracking section join attempts
```

Comment:
Can you add a jira ticket for this? (i.e. add the link here on github, not in the code itself)